### PR TITLE
Add script to migrate parameters

### PR DESCRIPTION
**Pull request by Tristan-WorkGH**

### CODE REVIEW

File: migration/migrate_parameters_from_study.sql [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    err_pg_exception_hint text;
+    err_pg_exception_context text;
+BEGIN
+    --lock table study in exclusive mode;
+    raise log 'Script starting at %', now();
+    raise log 'user=%, db=%, schema=%', current_user, current_database(), current_schema();
+
+    /* Case OPF: copy from db.study.<table> to db.<service>.<table> (easy, no problem to migrate)
+     * Case localdev & Azure: copy from study.public.<table> to <service>.public.<table> (need to copy between databases...)
+     *   [0A000] cross-database references are not implemented: "ref.to.remote.table"
```
Comment (SlimaneAmar) [inline migration/migrate_parameters_from_study.sql:41]: Don't undersand, cross-database it's not possible ?

Comment (Tristan-WorkGH) [reply migration/migrate_parameters_from_study.sql:67]: not without using an extension (a data wrapper) with what they call "foreign table" with an "foreign server"

**Review by ayolab — APPROVED** [review]
Test OK

**Review by SlimaneAmar — APPROVED** [review]

**Comment by Tristan-WorkGH** [comment]
Pin PR for security-analysis:
* gridsuite/study-server#502
* gridsuite/security-analysis-server#104

**Pull request merged**